.github/workflows/release.yml

```
@@ -0,0 +1,34 @@
+name: Release
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - v*
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        version: 2.6.x
+
+    - name: Build
+      run: |
+        printf -- "${GEM_PRIVATE_KEY}\n" > $HOME/certs/gem-private_key.pem
+        gem cert --add certs/gem-public_cert.pem
+        gem build kitchen-terraform.gemspec --silent
+      env:
+        GEM_PRIVATE_KEY: ${{ secrets.RUBYGEMS_PRIVATE_KEY }}
```

I think this key should be named consistently.

```suggestion
        GEM_PRIVATE_KEY: ${{ secrets.GEM_PRIVATE_KEY }}
```